src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Person } from '../../types/Person';
+import { peopleFromServer } from '../../data/people';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
+
+type Props = {
+  onSelect: (peoples: Person | null) => void;
+};
+
+export const Dropdown: React.FC<Props> = React.memo(({ onSelect }) => {
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [show, setShow] = useState(false);
+
+  const applyQuery = useCallback(debounce(setAppliedQuery, 300), []);
```

Comment:
Fix it 
<img width="954" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/6c2d6cca-47ea-4db3-b2a8-98f223f7d7a1">